test/functionsTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
File is named incorrectly, name of a file that contains a class should be identical to the class it contains so renaming this file to FunctionsTest.php is needed